app/Actions/Place/AddPlace/AddPlaceAction.php

```diff
@@ -115,6 +116,7 @@ function () use ($placeRequest) {
 
                 $this->placeRepository->setWorktime($place, $placeRequest->getWorktime());
 
+                Log::info("Place {$place->id} is added");
```

Comment:
add user id and name who performed action like `place: User 2 added place 1"`, place_add is event description